solver/edge.go

```diff
@@ -33,6 +34,28 @@ func newEdge(ed Edge, op activeOp, index *edgeIndex) *edge {
 		cacheRecords:       map[string]*CacheRecord{},
 		cacheRecordsLoaded: map[string]struct{}{},
 		index:              index,
+		debug:              debugScheduler,
+	}
+	if !e.debug && len(debugSchedulerSteps) > 0 {
+		withParents := strings.HasSuffix(debugSchedulerSteps[0], "^")
+		name := strings.TrimSuffix(debugSchedulerSteps[0], "^")
+		for _, v := range debugSchedulerSteps {
+			if strings.Contains(name, v) {
+				e.debug = true
+				break
+			}
+		}
+		if !e.debug && withParents {
+			for _, vtx := range ed.Vertex.Inputs() {
+				name := strings.TrimSuffix(vtx.Vertex.Name(), "^")
+				for _, v := range debugSchedulerSteps {
+					if strings.Contains(name, v) {
+						e.debug = true
+						break
+					}
+				}
+			}
+		}
```

Comment:
Do you have some examples?

iiuc it could smth like `BUILDKIT_SCHEDULER_DEBUG_STEPS=unpark,^status`?

Reply:
`BUILDKIT_SCHEDULER_DEBUG_STEPS=COPY foo bar^`